src/ConstructiveSolidGeometry/VolumePrimitives/Cone.jl

```diff
@@ -83,22 +83,31 @@ struct Cone{T,CO,TR,TP<:Union{Nothing,T}} <: AbstractVolumePrimitive{T, CO}
 end
 
 #Type conversion happens here
+function Cone{T,CO}(r, φ::Tuple{T,T}, hZ, origin, rotation) where {T,CO}
+    r = _csg_convert_args(T, r)
+    φ = _csg_convert_args(T, φ)
+    _φ = abs(φ[2]-φ[1])
+    rotation=rotation*SMatrix{3}(cos(φ[1]),sin(φ[1]),0,-sin(φ[1]),cos(φ[1]),0,0,0,1)
```

Comment:
Shorten this a bit? Something like `rotation *= SMatrix{3}(RotZ(φ[1]))`

Reply:
@SebastianRuffert  could you use `RotZ(φ[1])` instead of writing all elements?